doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -497,8 +491,17 @@ inferred from the whole context of the goal (see for example section
 Definitions
 ~~~~~~~~~~~
 
-.. tacn:: pose
-   :name: pose (ssreflect)
+.. tacn:: pose fix @ssrbvar {* @ssrbinder } {? %{ struct @ident %} } @ssrfwd
+          pose cofix @ssrbvar {* @ssrbinder } @ssrfwd
+          pose @ident {* @ssrbinder } @ssrfwd
+   :name: pose (ssreflect); _; _
+
+   .. insertprodn ssrfwd ssrsetfwd
+
+   .. prodn::
+      ssrfwd ::= {? : @term } := @term
```

Comment:
I think we should rename this non terminal to `ssrdefbody` or something along the lines.